deploy/scripts/environment_check.sh

```diff
@@ -100,10 +114,54 @@ validate_ds() {
   fi
 }
 
-dependencies kubectl jq mktemp
+check_package_installed() {
+  local packages=($@)
+  local pods=$(kubectl get pods -o name | grep longhorn-environment-check)
+
+  local allFound=true
+
+  for ((i=0; i<${#packages[@]}; i++)); do
+    local package=${packages[$i]}
+
+    for pod in ${pods}; do
+      installed=`kubectl exec -it $pod -- nsenter --mount=/proc/1/ns/mnt -- bash -c "dpkg --get-selections ${package}" | grep -c 'install'`
```

Comment:
This should also work for all distros we support in this script.

Reply:
Oops, sorry, forgot to handle `dpkg ...`